lib/highlighter.rb

```diff
@@ -11,6 +12,11 @@ def registered(app)
 
   module Helpers
     def _highlight(string, language, class_name=nil)
+      error_message = "Code blocks must be fenced with proper language designated. If you don't know what language to use, specify text.\n\n"
```

Comment:
"If you don't know what language to use, specify text" -> If you don't know what language to use, specify ```text